composer.json

```diff
@@ -28,13 +28,13 @@
         "source": "https://github.com/cvo-technologies/cakephp-gearman"
     },
     "require": {
+        "php": ">=5.5.9",
         "ext-gearman": "*",
         "cakephp/cakephp": "~3.2"
     },
     "require-dev": {
-        "cakephp/cakephp-codesniffer": "2.*",
```

Comment:
Ah that does make sense, `require-dev` dependencies are not included from dependencies though.